pages/_/slate.js

```diff
@@ -152,7 +154,10 @@ export default class SlatePage extends React.Component {
       <WebsitePrototypeWrapper title={title} description={body} url={url} image={image}>
         <div css={STYLES_ROOT}>
           <WebsitePrototypeHeaderGeneric href={headerURL} title={headerTitle}>
-            <ProcessedText text={this.props.slate.data.body} />
+            <ViewAll maxCharacter={208}>
+              {/* <ProcessedText text={this.props.slate.data.body} /> */}
```

Comment:
You should still capture what `<ProcessedText />` does, that function transforms links into real links.